src/users/users.ts

```diff
@@ -298,12 +298,25 @@ export class Users {
     };
   }
 
-  async listAuthFactors(payload: ListAuthFactorsOptions): Promise<Factor[]> {
-    const { data } = await this.workos.get<FactorResponse[]>(
-      `/users/${payload.userId}/auth/factors`,
+  async listAuthFactors(
+    options: ListAuthFactorsOptions,
+  ): Promise<AutoPaginatable<Factor>> {
+    return new AutoPaginatable(
```

Comment:
What was the issue here? Was it not paginated before?

Reply:
Yeah... The API output is paginated.